packages/api/models/schemas.js

```diff
@@ -44,7 +44,7 @@ module.exports.asyncOperation = {
     description: { type: 'string' },
     operationType: {
       type: 'string',
-      enum: ['Migration Count Report', 'ES Index', 'Bulk Granules', 'Bulk Granule Delete', 'Bulk Granule Reingest', 'Kinesis Replay', 'Reconciliation Report'],
+      enum: ['Data Migration 2', 'Migration Count Report', 'ES Index', 'Bulk Granules', 'Bulk Granule Delete', 'Bulk Granule Reingest', 'Kinesis Replay', 'Reconciliation Report'],
```

Comment:
See comment above re: Data Migration 2/Data Migration

Reply:
> Nit: Should this just be 'Data Migration' if we're only doing one type of Data Migration async operation?


Yeah, I think this is a reasonable call out. I was okay with `Data Migration 2` since the async operation points to that lambda though I don't think that's good justification for leaving it. I think `Data Migration` is cleaner as well and `2` in the name is possibly even more confusing.

Reply:
[`e0b53d0` (#2165)](https://github.com/nasa/cumulus/pull/2165/commits/e0b53d0caad6c024a490901ccc2670b669500cb5)